### PR TITLE
Wikipedia Scraping

### PR DESCRIPTION
**Pull request by Abhay-1552**
BeautifulSoup and Flask Framework

### CODE REVIEW

File: ThirdAI/Terms and Conditions/third_AI.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+from thirdai import licensing, neural_db as ndb
```
Comment (NitkarshChourasia) [inline ThirdAI/Terms and Conditions/third_AI.py:1]: Use docstring and comments, to make the program easily understandable to others as well as you.

File: ThirdAI/Terms and Conditions/tkinter_UI.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+query_entry = tk.Entry(root, font=customsize(20), width=30)
+query_entry.place(x=70, y=300)
+
+processing = Button(root, text="Processing", font=15, fg='black', bg="grey", width=10, command=processing)
+processing.place(x=245, y=355)
+
+clear = Button(root, text="Clear", font=15, fg='black', bg="grey", width=10, command=clear_all)
+clear.place(x=245, y=405)
+
```
Comment (NitkarshChourasia) [inline ThirdAI/Terms and Conditions/tkinter_UI.py:107]: Make it __OOP__ based and __KISS__ based.
```
if __name__ == "__main__":
        main()
```

File: Wikipdedia/practice_beautifulsoap.py [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+def lang():
+    try:
+        response = requests.get("https://www.wikipedia.org/")
+        response.raise_for_status()
+        soup = BeautifulSoup(response.content, 'html.parser')
+
+        for option in soup.find_all('option'):
+            language = option.text
+            symbol = option['lang']
+            language_symbols[language] = symbol
```
Comment (NitkarshChourasia) [inline Wikipdedia/practice_beautifulsoap.py:16]: Also, provide a proper README.md file, for other people, general people to understand what is this project is all about.
and cover basic aspects like:
- How to install
- How to uninstall
- How to use
etc...,
What are the key features, what are bugs, what to improve, what can be further refactored, etc..., just explaining your project in full length.

File: Wikipdedia/template/output.html [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+        <h6> in {{ language }} language</h6>
+    </div>
+    <div class="container mt-4">
+        <div class="container border p-3">
+            <pre>{{ data }}</pre>
+        </div>
+    </div>
+
+    <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.0.7/dist/umd/popper.min.js"></script>
```
Comment (NitkarshChourasia) [inline Wikipdedia/template/output.html:31]: Documentation and OOP based and KISS approach is missing everywhere.

# MOST IMPORTANT:
README.md 
and 
Documentation is missing.

**Review by NitkarshChourasia — CHANGES_REQUESTED** [review]

**Review by rein14 — APPROVED** [review]
Looks good

**Review by NitkarshChourasia — APPROVED** [review]
Looks good and promising.

**Pull request merged**